tools/packaging/kernel/configs/fragments/x86_64/sev/sev.conf

```diff
@@ -1,12 +1,9 @@
 # AMD Secure Encrypted Virtualization (SEV)
- 
+
 CONFIG_AMD_MEM_ENCRYPT=y
-CONFIG_AMD_MEM_ENCRYPT_ACTIVE_BY_DEFAULT=y
-CONFIG_CRYPTO_DEV_SP_PSP=y
-CONFIG_CRYPTO_DEV_CCP=y
-CONFIG_SECURITYFS=y
```

Comment:
I understand the CRYPTO settings are for the host and not needed in the guest; good.

What about CONFIG_SECURITYFS? Our efi_secret module (`CONFIG_EFI_SECRET=m`) uses it. Is it enabled in some other place?

Reply:
@dubek I was attempting to cleanup the sev.conf by removing all unnecessary CONFIG options. I confirmed that the list I currently provide in this PR is the minimal set, and I successfully booted an SEV kata container using a 5.19 guest. I'm using the command `./build-kernel.sh -x sev setup` to generate the .config. CONFIG_SECURITYFS is making it's way into the final .config. I'm not sure if something else is adding it, or it's there by default now? I'm happy to undo it's removal in this PR if you feel it necessary.

Reply:
```
root@rysavino-ubuntu-02:~/kata-containers/tools/packaging/kernel/configs# grep -R "CONFIG_SECURITYFS" .
./arm64_kata_kvm_4.14.x:# CONFIG_SECURITYFS is not set
./arm64_kata_kvm_virtio-fs-v0.3.x:# CONFIG_SECURITYFS is not set
./powerpc_kata_kvm_4.19.x:# CONFIG_SECURITYFS is not set
./arm64_kata_kvm_4.19.x:# CONFIG_SECURITYFS is not set
./s390_kata_kvm_5.4.x:CONFIG_SECURITYFS=y
./fragments/x86_64/.config:CONFIG_SECURITYFS=y
./arm64_kata_kvm_5.4.x:# CONFIG_SECURITYFS is not set
./powerpc_kata_kvm_5.4.x:# CONFIG_SECURITYFS is not set
./x86_64_kata_kvm_4.14.x:# CONFIG_SECURITYFS is not set
./s390_kata_kvm_4.19.x:# CONFIG_SECURITYFS is not set
./powerpc_kata_kvm_4.14.x:# CONFIG_SECURITYFS is not set
```

Maybe default option? From above search it's making its way into fragments/x86_64/.config, but isn't being set in any conf that would affect x86_64.

Reply:
If it's in the final `.config` I think it's fine for now (maybe CONFIG_EFI_SECRET selects it, I need to look it up).